maud/src/lib.rs

```diff
@@ -189,3 +190,17 @@ mod rocket_support {
         }
     }
 }
+
+#[cfg(feature = "actix-web")]
+mod actix_support {
+    use PreEscaped;
+    use actix_web::{Responder, HttpResponse, HttpRequest, Error};
+
+    impl Responder for PreEscaped<String> {
+        type Item = HttpResponse;
+        type Error = Error;
+        fn respond_to<String>(self, _req: &HttpRequest<String>) -> Result<Self::Item, Self::Error> {
```

Comment:
This is better written as 

```rust
fn respond_to<S>(self, _req: &HttpRequest<S>) -> ...
```

to avoid confusion with the built-in `String` type